prep-javascript-for-loops/main.js

```diff
@@ -0,0 +1,5 @@
+// create your loops here.
+
+for (let i = 100; i >= 1; i--) {
+  console.log('Time till explosion', i + '!');
+}
```

Comment:
Hi Aaron, please include all three loops in your solution.

Reply:
Ok, sorry about that.